spark/spark-adapter-api/src/main/scala/za/co/absa/spline/spark/adapter/URIPrefixes.scala

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-let _ = require('lodash')
+package za.co.absa.spline.spark.adapter
 
-let getDetails = function (data, nodeId) {
-    return _.find(data, function (details) { return details.mainProps.id === nodeId })
+object URIPrefixes {
+  //prefix used in identifiers for saveAsTable writes
+  val managedTablePrefix = "table://"
```

Comment:
Is this one used somewhere ?

Reply:
yes, it's used in adapters, `SaveAsTableCommandParserImpl` class